cypress/integration/conditions/boolean_conditions.cy.js

```diff
@@ -34,58 +34,108 @@ const setUpTable = () => {
     expectTableToBeVisible()
 }
 
-const addConditions = (conditions) => {
+const addConditions = (conditions, dimensionName) => {
     cy.getBySelLike('layout-chip').contains(dimensionName).click()
     conditions.forEach((conditionName) => {
-        cy.getWithDataTest('{conditions-modal-content}')
+        cy.getBySel('conditions-modal-content')
             .findWithDataTest('{dhis2-uicore-checkbox}')
             .contains(conditionName)
             .click()
     })
-    cy.getWithDataTest('{conditions-modal}').contains('Update').click()
+    cy.getBySel('conditions-modal').contains('Update').click()
 }
 
-const assertTooltipContainsEntries = (entries) => {
-    entries.forEach((entry) =>
-        cy.getWithDataTest('{tooltip-content}').contains(entry)
-    )
-}
+const assertTooltipContainsEntries = (entries) =>
+    entries.forEach((entry) => cy.getBySel('tooltip-content').contains(entry))
+
+const assertChipContainsText = (suffix) =>
+    cy.getBySelLike('layout-chip').contains(suffix).trigger('mouseover')
+
+describe('boolean conditions - Yes/NA', () => {
+    const dimensionName = TEST_DIM_YESONLY
 
-describe('boolean conditions', () => {
     beforeEach(() => {
         cy.visit('/', EXTENDED_TIMEOUT)
-        setUpTable()
+        setUpTable(dimensionName)
     })
 
     it('Yes selected', () => {
-        addConditions(['Yes'])
+        addConditions(['Yes'], dimensionName)
+
+        expectTableToMatchRows([`${getCurrentYearStr()}-01-01`])
+
+        assertChipContainsText(`${dimensionName}: 1 condition`)
+
+        assertTooltipContainsEntries([stageName, /\bYes\b/])
+    })
+
+    it('Not answered selected', () => {
+        addConditions(['Not answered'], dimensionName)
 
         expectTableToMatchRows([
             `${getCurrentYearStr()}-01-01`,
```

Comment:
Very minor: what about grabbing the year once, then reusing it throughout the tests? `const currentYear = getCurrentYearStr()`